modules/nw-egress-ips-automatic.adoc

```diff
@@ -45,6 +45,12 @@ $ oc patch netnamespace project1 --type=merge -p \
 $ oc patch netnamespace project2 --type=merge -p \
   '{"egressIPs": ["192.168.1.101"]}'
 ----
+
+[NOTE]
+====
+Because OpenShift SDN manages the `NetNamespace` object, you can make changes only by modifying the existing `NetNamespace` object. Do not create a new `NetNamespace` object.
+====
+
 +
```

Comment:
This `+` in the line following the Note should be preceding it - I think once you fix that it will fix the rendering issue for steps following this one :)

Reply:
Done. Thanks! All rendering just fine.